app/models/entitlement.rb

```diff
@@ -59,12 +66,20 @@ def set_belongsto_filters(filter)
   end
   alias belongsto_filter= set_belongsto_filters
 
+  def update_managed_filters_on_name_change(old_tag, new_tag)
+    return if !filters.present? || !filters.key?("managed") || !filters["managed"].present?
```

Comment:
Slightly simpler to read

```suggestion
    return if filters.blank? || !filters.key?("managed") || filters["managed"].blank?
```

Reply:
Ok, pushed.  💚